Project/Assets/ML-Agents/Examples/Match3/Scripts/Match3Board.cs

```diff
@@ -41,6 +62,30 @@ void Start()
             InitRandom();
         }
 
+        public override BoardSize GetMaxBoardSize()
+        {
+            return new BoardSize
+            {
+                Rows = MaxRows,
+                Columns = MaxColumns,
+                NumCellTypes = NumCellTypes,
+                NumSpecialTypes = NumSpecialTypes
+            };
+        }
+
+        public override BoardSize GetCurrentBoardSize()
+        {
+            return m_CurrentBoardSize;
+        }
+
+        public void UpdateCurrentBoardSize()
```

Comment:
maybe just add a comment to remind people that this is an example, and they can update their board size however they want.